src/main/java/seedu/duke/DukeException.java

```diff
@@ -0,0 +1,16 @@
+package seedu.duke;
+
+public class DukeException extends Exception{
+
+    public DukeException(String message) {
+        super(message);
+    }
+
+
+
+//    public static void main(String[] args) {
+//        try {
+//
+//        }
+//    }

```

Comment:
Will it be more reader-friendly for other reviewers if commented out code is removed instead? Similar questions for other parts of the code